webrtc.html

```diff
@@ -9495,7 +9495,16 @@ <h3>Introduction</h3>
       queries can be linked by the <code><a>RTCStats</a></code> <a data-for=
       "RTCStats">id</a> dictionary member. Thus, a Web application can make
       measurements over a given time period by requesting measurements at the
-      beginning and end of that period.</p>
+        beginning and end of that period.</p>
+      <p>
+        Stats objects may have a limited lifetime. Until the end of
+        their lifetime, they are always present in the result from
+        getStats(). When their lifetime ends, a record of the
+        statistics for that object is emitted through a
+        "statslifetimeended" event, containing an RTCStats
```

Comment:
I'm fond of algorithms. E.g. *"When their lifetime ends, the UA MUST queue a task to run the following steps:"*

Reply:
Makes sense.

Reply:
Added the algorithm under the "onstatslifetimeended" handler. Looked to me like a more natural place than in an introduction section.
Have to make sure I'm phrasing it right, though.